02_working_with_strings/translate.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2015 Ami. All rights reserved.
+
+def user_input():
```

Comment:
Refactor this function to only return the user input, and translate it in a different function.